src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -2957,6 +2974,10 @@ public static boolean paxoTopologyRepairStrictEachQuorum()
         return conf.paxos_topology_repair_strict_each_quorum;
     }
 
+    // TODO imperative that at startup we check that the legacy paxos strategy is compatible
+    // with cluster metadata for consensus migration
+    // If another node updates the CM with consensus migration and we don't support it that is a serious issue as well
+    // Ideally this config could only ever exist in CM so that the cluster always agrees on what it is
```

Comment:
Agree re LWT, `paxos` doesn't make sense because it doesn't actually mean `paxos` it means `paxos_and_accord` which is why for now it is called `migration`. WDYT would be better than `migration`?

This config option is not intended to be a starting state. It's intended to allow you to prevent accidental Paxos usage once you have migrated to Accord by switching from `migration` to `accord`.

Reply:
TransactionalStrategy would cover everything I guess?

Reply:
This PR is out of date now that I have a new rebased branch elsewhere. Just FYI.